tools/fiptool/fiptool.c

```diff
@@ -561,10 +570,29 @@ static void parse_plat_toc_flags(char *arg, unsigned long long *toc_flags)
 	*toc_flags |= flags << 32;
 }
 
+static int is_power_of_2(unsigned long x)
+{
+        return x && !(x & (x - 1));
+}
```

Comment:
Sorry, I didn't mean to suggest sharing this header with the tools - that was my mistake. It's fine as-is then.